src/lib/core/style/_menu-common.scss

```diff
@@ -42,4 +42,12 @@ $md-menu-vertical-padding: 8px !default;
   &[disabled] {
     cursor: default;
   }
+
+  md-icon {
```

Comment:
No, as far as I could tell, no exact measurement for this case in the spec. This was the value in Material 1.